### DISCUSSION TITLE
Keep signing out

### DISCUSSION THREAD

**Discussion by Ronaldes10** [redphx/better-xcloud#499] [General]
### Checklist

- [X] I will only use English in my report.
- [X] I have used the search function for [**open and closed issues**](https://github.com/redphx/better-xcloud/issues?q=is%3Aissue) to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### Questions

- [X] xCloud officially supports my country/region.
- [ ] The bug doesn't happen when I disable Better xCloud script.
- [X] The bug didn't happen in previous Better xCloud version (name which one).

### Device

Other

### Device

Windows 10

### Operating System

Windows 10

### Android app/Browser Version

Chrome 128.0

### Better xCloud Version

5.7.8

### Game list

I CANNOT BUT PLAY ANY GAME!

### Reproduction Steps

I just logged out of my account, and when I came back, I couldn't access it anymore.

**Comment by Ronaldes10** [comment]
Red. HELP ME!!!

**Comment by btncua** [comment]
Please relax, you won't solve anything by panicking. What message do you see when you try to sign in to your xbox account?

**Reply by Ronaldes10** [reply]
It just disconnects me. That's all. I haven't accessed xcloud for three days, and I don't know what to do anymore!

**Reply by btncua** [reply]
@Ronaldes10 share video or screenshot

**Comment by Ronaldes10** [comment]
https://github.com/user-attachments/assets/ee723106-d1c4-4817-afff-ec7df436de5f

The video will be in Brazilian Portuguese

**Reply by Ronaldes10** [reply]
And now what should I do?

**Comment by redphx** [comment]
Try incognito mode or another browser. If it still doesn't work, ask Xbox support.
  
This is not caused by Better xCloud.

**Reply by Ronaldes10** [reply]
But, to access support do you need an Xbox account? If so, that's a problem! And also Xbox doesn't answer my emails, but I'll try to do something I can do for the account. Thank you
